scripts/qa_crowbarsetup.sh

```diff
@@ -2407,6 +2407,9 @@ function custom_configuration()
                 proposal_set_value rabbitmq default "['attributes']['rabbitmq']['ha']['storage']['drbd']['size']" "$drbd_rabbitmq_size"
                 proposal_set_value rabbitmq default "['deployment']['rabbitmq']['elements']['rabbitmq-server']" "['cluster:$clusternamedata']"
             fi
+            if iscloudver 5plus; then
+                proposal_set_value rabbitmq default "['attributes']['trove']['enabled']" true
```

Comment:
test failed with
-e:6:in `<main>': undefined method `[]=' for nil:NilClass (NoMethodError)
+ complain 30 'rubyjsonparse 
        j['\''attributes'\'']['\''trove'\'']['\''enabled'\'']=true
        puts JSON.pretty_generate(j)